contracts/ligo/src/english_auction_tez.mligo

```diff
@@ -42,6 +43,7 @@ type auction_entrypoints =
   | Bid of nat
   | Cancel of nat
   | Resolve of nat
+  | Change_admin of address
```

Comment:
Consider using two-phased admin change.
This issue came from security audit of some other contracts:
It is possible to lock-out contract administrator by changing to a dead address.
Two-phase admin change would require a new proposed admin to confirm the change.
See [simple admin](https://github.com/tqtezos/smart-contracts/blob/master/shared/fa2_modules/simple_admin.mligo) for example